cypress/integration/parallel-2/membershipSave.spec.ts

```diff
@@ -9,6 +9,18 @@ import { signInAndAcceptCookies } from '../../lib/signInAndAcceptCookies';
 
 if (featureSwitches.membershipSave) {
 	describe('Cancel membership saves', () => {
+		const setSignInStatus = () => {
+			cy.window().then((window) => {
+				// @ts-ignore
```

Comment:
Do we need this comment/ts-ignore?

Reply:
I think we could add the same config from `shared/globals.ts` to `cypress/support/index.ts` (or similar) but I don't know if it's worth it